### ISSUE TITLE
Unable to find thing with id: DateTimeDevice

### ISSUE THREAD

**Issue by flatsiedatsie** [tomasy/date-time-adapter#17]
Saw this error in the logs:

```
2019-11-24 09:10:07.069 INFO   : date-time: util.py:89 INFO CALC_SUNRISE today.utc: 2019/11/24 08:10:07 sunrise: 2019/11/25 05:43:13 sunrise_local: 2019-11-25 05:43:13.396697+00:00
2019-11-24 09:10:07.073 INFO   : Unhandled Rejection
2019-11-24 09:10:07.075 ERROR  : Error: Unable to find thing with id: DateTimeDevice
    at /home/pi/mozilla-iot/gateway/build/webpack:/src/models/things.js:267:1
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
2019-11-24 09:10:07.077 INFO   : date-time: util.py:89 INFO CALC_SUNRISE today.utc: 2019/11/24 08:10:07 sunrise: 2019/11/25 05:43:13 sunrise_local: 2019-11-25 05:43:13.396697+00:00
2019-11-24 09:10:07.077 INFO   : date-time: util.py:96 INFO New sunrise 2019-11-25 05:43:13.396697+00:00

```

The thing has not been added to the things overview yet, so it's probably an error relating to that.

**Comment by tomasy** [comment]
It seems like the DateTime adapter has been loaded. The output with "date-time " indicate this but the gateway has not recognized it.
Have you tried to reinstall the adapter? Have it worked before?

**Comment by flatsiedatsie** [comment]
I have not tried to reinstall it. I haven't even started using the adapter yet at that point - I hadn't even added the thing yet.

**Comment by mrstegeman** [comment]
This was a gateway issue, fixed by mozilla-iot/gateway#2294.
